src/image_builder.py

```diff
@@ -110,6 +110,16 @@ def image_builder(buildspec):
                 labels[var] = file_name
                 labels[f"{var}_URI"] = uri
 
+        if str(BUILDSPEC["framework"]).startswith("huggingface"):
+            if "transformers_version" in image_config:
```

Comment:
Is `image_config` definitely a dict, or is it an object of a custom class?

Reply:
It's a CommentedMap rather. But it also supports __contains__()